src/portfolios/portfolio/components/shared/PortfolioDrawer.vue

```diff
@@ -259,7 +260,9 @@ export default class PortfolioDrawer extends Vue {
   }
 
   public saveDescription(): void {
-    PortfolioStore.setPortfolioData(this.portfolio);
+    if(hasChanges(this.portfolioDescription, this.portfolio.description)) {
+      PortfolioStore.updatePortfolioDescription(this.portfolio.description);
```

Comment:
please update `this.portfolioDescription` to the value of `this.portfolio.description` to prevent unneeded  saves when tabbing through after changing the value
`this.portfolioDescription = this.portfolio.description;`